### PR TITLE
Create copyright

### PR DESCRIPTION
**Pull request by sravyamks**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build was run locally and any changes were pushed
- [ ] Lint has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: sdb
+Source: https://github.com/mono/sdb
```
Comment (prakashsurya) [inline debian/copyright:3]: "delphix/sdb" is not related to "mono/sdb"

**Comment by sdimitro** [comment]
Are we adding copyrights to our the debian package that mark it as MIT? `sdb` is not MIT licensed.

I'm sorry if I missed something but why do we need this change?

**Comment by sravyamks** [comment]
Product wants to make a list of all licenses included for the open-source packages we install on Delphix engine. When using a script to retrieve this information, There are some packages installed without copyright files. I was trying to find the source package and add the copyright file from there. For example in https://github.com/delphix/bpftrace/pull/8 - I took the copyright file from http://changelogs.ubuntu.com/changelogs/pool/universe/b/bpftrace/bpftrace_0.9.2-1/copyright.
Didn't realize sdb did not have an upstream mentioned and took it from ubuntu packages llike the other one. Any idea what would be the right place to get the original copyright file for this package.

**Comment by sebroy** [comment]
@sravyamks The open source license associated with this repo is contained in the LICENSE file at the top of the repo. The LICENSE file is a best practice that we have across all of our delphix org github repos. If the requirement is to include license information (and not copyright information), then do we not already have what we need?

**Pull request closed without merging**